frontend/src/general/dialog/__tests__/Dialog.test.jsx

```diff
@@ -23,12 +24,33 @@ describe("<Dialog/> Class Component", () => {
   );
 
   describe("render() function", () => {
-    it("returns a component that contains the right elements", () => {
+    it("if noCancel prop is default to false, returns a component that contains the right elements", () => {
```

Comment:
Add a describe("renders a component with the expected elements", ()=>{
it("when noCancel is true", ()=>{ // test here})
it("when noCancel is false", ()=>{ // test here})
})